qutebrowser/test/utils/debug/test_qenum_key.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for qutebrowser.utils.debug.qenum_key."""
+
+import pytest
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QStyle, QFrame
+
+from qutebrowser.utils import debug
+
+
+def test_no_metaobj():
+    """Test with an enum with no meta-object."""
+
+    with pytest.raises(AttributeError):
+        # Make sure it doesn't have a meta object
+        # pylint: disable=pointless-statement,no-member
+        QStyle.PrimitiveElement.staticMetaObject
```

Comment:
I just noticed a `assert hasattr(QStyle.PrimitiveElement, 'staticMetaObject')` would probably be much clearer here and in `test_metaobj()`.